packages/excalidraw/clipboard.ts

```diff
@@ -319,9 +332,21 @@ const parseClipboardEvent = async (
       return mixedContent;
     }
 
+    const imageType = event.clipboardData?.types[0]?.startsWith("image/") ? event.clipboardData?.types[0] : "image/png";
```

Comment:
Similar to the above comments. Should we assume all clipboard items are of the same type? Should we default to png?